ui/app/src/services/configuration.ts

```diff
@@ -62,14 +69,14 @@ const LegacyPanelIdMap: any = {
   'ice-tools-panel': 'craftercms.ice.ice',
   'component-panel': 'craftercms.ice.components',
   'medium-panel': 'craftercms.ice.simulator',
-  'targeting': 'craftercms.ice.audiences'
+  targeting: 'craftercms.ice.audiences'
```

Comment:
let’s talk about removing all these mappings, being a brand new config, we don’t need to map legacy stuff